window/src/os/macos/window.rs

```diff
@@ -1791,7 +1791,8 @@ impl WindowView {
                 "\x08"
             } else if virtual_key == super::keycodes::kVK_Tab {
                 "\t"
-            } else if !use_ime && virtual_key == super::keycodes::kVK_Delete {
+            } else if virtual_key == super::keycodes::kVK_Delete {
+                // When using IME in macOS, delete applied in candidate window.
```

Comment:
This appears to be a different issue from the main thing you mentioned in #1409; could you give some more details on how to reproduce the issue that you're fixing with this?

Reply:
ah, I see now; I'll fix this in a commit I have in progress!